source/base/parameter_handler.cc

```diff
@@ -519,17 +519,25 @@ ParameterHandler::parse_input(const std::string &filename,
       std::istream is(&fb);
       std::string file_ending = filename.substr(filename.find_last_of('.') + 1);
       boost::algorithm::to_lower(file_ending);
-      if (file_ending == "prm")
```

Comment:
I would like to have this if-statement here, because we will later reintroduce it anyway but it now helps understanding the logic of this function. The original intention of this function otherwise gets blurred or we would have to add documentation